modules/reindex/build.gradle

```diff
@@ -26,6 +26,7 @@ integTest {
   cluster {
     // Whitelist reindexing from the local node so we can test it.
     setting 'reindex.remote.whitelist', 'myself'
+    plugin 'optional-auth', project(':test:plugins:optional-auth')
```

Comment:
This approach to testing this works with Elasticsearch's build but I'm not sure what the right way to communicate it to the client builds is. Or even if that is possible or required.